sdk/formrecognizer/test-resources.json

```diff
@@ -11,7 +11,7 @@
         },
         "location": {
             "type": "string",
-            "defaultValue": "westcentralus",
```

Comment:
We need FR to run in `westcentralus`. Where will be the right place to set this then

Reply:
@danieljurek

Reply:
Send [PR ](https://github.com/Azure/azure-sdk-for-net/pull/12862)to set it back and we will consider to control its vaule in tests.yml.

Reply:
Thank you!